src/mono/mono/mini/mini-runtime.c

```diff
@@ -4351,7 +4351,6 @@ init_class (MonoClass *klass)
 
 	const char *name = m_class_get_name (klass);
 
-#ifdef TARGET_AMD64
```

Comment:
Shouldn't this be TARGET_AMD64 or TARGET_ARM64 ?

Reply:
Right.